### ISSUE TITLE
Make an option to host snapshots only and make those default for warp-barrier flagged nodes.

### ISSUE THREAD

**Issue by Emperornero** [paritytech/parity#8592]
> - **Which Parity version?**: 1.11.0
> - **Which operating system?**: Windows 
> - **How installed?**: via installer 
> - **Are you fully synchronized?**: no 
> - **Which network are you connected to?**: ethereum 
> - **Did you try to restart the node?**: yes

Running with

```
parity --warp-barrier 5400000  --snapshot-peers=50 --max-peers=50 --mode=active --no-ancient-blocks --pruning=fast --pruning-history=8  --pruning-memory=32 --db-compaction="hdd" --ws-interface 192.168.1.7 --jsonrpc-interface 192.168.1.7 --ui-interface 192.168.1.7 --ws-hosts all --ws-origins all
pause
```

Cannot get snapshot going even after DB Kill.  It simply defaults to normal sync.

I think I've tried around 20 times now.

Can we make a node that only serves snapshots?  Warp has been amazingly hit or miss as of lately and it's like a lottery actually connecting to a client that can serve snapshots.  This seems like it would be a huge quality of life improvement and I would gladly run one of these nodes.

EDIT: So I've gone back and counted how long it took me doing parity db kill and deleting the cache and chains folders before I started finally warp syncing and I did it a grand total of 37 times.

Let me reiterate.

**THIRTY SEVEN TIMES KILLING THE DB BEFORE I GOT A SNAPSHOT PEER**

Make an option to host "snapshot only" nodes along side a normal node and end this lottery warp sync garbage.

**Comment by Tbaut** [comment]
It seems that the `5400000` was simply too high as the creation of a snapshot takes a lot of time now. however, it shouldn't get back to normal sync. A solution got merged 2 days ago https://github.com/paritytech/parity/pull/8543, it will most probably get fixed in 1.11.1 next week. stay tuned. You can also follow https://github.com/paritytech/parity/pull/8544

**Comment by ngotchac** [comment]
An issue solved in #8555 was also the root of slow snapshot restoration.
